app/views/api/metrics/index.html.slim

```diff
@@ -13,7 +13,7 @@ section.Section.u-toggleableBySettingsBox data-state="open"
     ' .
     - if show_mappings?
       ' A method needs to be mapped to one or more URL patterns in the
-      => link_to 'Mapping Rules', edit_admin_service_integration_path(@service, anchor: 'mapping-rules'), data: { behavior: 'open-mapping-rules' }
+      => link_to_mapping_rules(service: @service)
```

Comment:
Isn't this link also used when independent_mapping_rules are disabled?

Reply:
never mind, you added the anchor to the helper. problem is it doesn't need the anchor when independent_mapping_rules are enabled